recipe/meta.yaml

```diff
@@ -10,6 +10,8 @@ source:
   sha256: 956afdeb9b5600e873326e410e9379684dac8f8f47ea569151a417984e7799cf
   patches:
     - 11525.patch
+    # https://github.com/numpy/numpy/issues/11460
+    - backport_dce7f20.patch
```

Comment:
@jjhelmus and @msarahan different from https://github.com/conda-forge/numpy-feedstock/pull/99/files#r207019449 this patch is not essential b/c it only fix testing, `numpy` is OK without it.